services/httpd/handler.go

```diff
@@ -117,14 +118,6 @@ func NewHandler(requireAuthentication, loggingEnabled, writeTrace, JSONWriteEnab
 			"ping-head",
 			"HEAD", "/ping", true, true, h.servePing,
 		},
-		route{ // Ping w/ status
```

Comment:
Hmm.  I think we should keep these for now and log a warning that they are deprecated.   We should log a message that they should use `/ping` instead.  Any load balancer configured to point to these is going to think the host is down and i won't be easy to determine why.